src/main/webapp/management-console.js

```diff
@@ -5,8 +5,10 @@ angular.module('managementConsole', [
   'gridshore.c3js.chart',
   'ui.router',
   'ui.bootstrap',
-  'LocalStorageModule',
-]).directive('fileModel', ['$parse', function ($parse) {
+  'LocalStorageModule'
+]).constant('CONSTANTS', {
+  'NO_BASE_CONFIGURATION_TEMPLATE': 'NONE - no base template'
```

Comment:
If you do not mind let's use ```<none>``` as I already have check mechanisms in place to make sure that something is selected before proceeding to next screen. We can fine tune this for 8.3